### DISCUSSION TITLE
error: No commits found. Change commits range, initial depth or use --ignore-empty to allow empty patch sets

### DISCUSSION THREAD

**Discussion by madsh93** [nuxt-community/sentry-module#365] [Q&A] [answered]
@nuxtjs/sentry: 5.1.4
nuxt: 2.15.7

### Sentry configuration
````
  sentry: {
    dsn: process.env.SENTRY_DNS,
    lazy: false,
    sourceMapStyle: 'hidden-source-map',
    disabled: isDev,
    publishRelease: {
      setCommits: {
        auto: true
      }
    }
  },
````

### Steps to reproduce
Deploy on Vercel

### What is Expected?
No errors

### What is actually happening?
I get following error: 
![image](https://user-images.githubusercontent.com/8860723/139510105-a58c5c7e-01e0-4050-b676-6f8fa10803b7.png)

I can bypass the error with `ignoreEmpty: true`

**Comment by rchl** [comment]
There is probably something about your git history that makes sentry-cli fail on it. It tries to find the history range from the last release to the specified commit. Maybe there is no previous release?

**Comment by madsh93** [comment, marked as answer]
Sorry! This was my own mistake. It seems my Github was not connected to Sentry. This article explains how to fix:

https://charles-stover.medium.com/how-i-debugged-sentry-release-no-commits-found-change-commits-range-d390ba4e326c

**Reply by tatyree** [reply]
Paywalled.

**Reply by quisido** [reply]
You can get around the Medium paywall by using incognito mode. Here are the steps nonetheless:

* Go to Sentry.
* Go to organization settings.
* Go to Integrations.
* Go to the GitHub integration.
* Edit the GitHub configurations.
* There may be a step here to choose which GitHub account to configure. If so, choose the owner of the repository in question.
* Use the "Add repository" dropdown to add the target repository.
